docs/tutorial/index.md

```diff
@@ -316,32 +169,27 @@ Open a new terminal window and run `ubuntu2204.exe`. Landscape server and relate
 The install link below will work only if you're logged in to the Microsoft Store with an account for which access to the app has been enabled.
 ```
 
-Time to install UP4W! Click on [this link](https://www.microsoft.com/store/productId/9PD1WZNBDXKZ), then on the big blue **Install** button.
+To install UP4W go to [this link to the Microsoft Store](https://www.microsoft.com/store/productId/9PD1WZNBDXKZ) and click **Install**.
 
 ![Install Ubuntu Pro for WSL from the Store](./assets/store.png)
 
+After installation has finished click **Start** to begin configuring UP4W.
 
-Once the installation is complete, you will see a **Start** button. Click it. That will start the UP4W Windows application. We'll use it to configure UP4W at next.
+% :NOTE: Instead of the GUI, it's possible to configure UP4W through the Windows registry, which enables you to do things at scale. This should be explained here or elsewhere.
```

Comment:
By commenting this out we remove the only place in this tutorial where we call the attention to the sysadmins that it's possible to do things at scale.

When I first wrote this tutorial, instead of using the GUI, I used the registry because I wanted to highlight that possibility.

Is there a better place, maybe earlier in this document, where we could emphasize that, even though this doc follows the GUI-workflow? Or, instead, you believe it's better to write a new document solely focused on the sysadmin? Per my understanding the ideal way to write a tutorial for sysadmins would require deploying the applications as they would (using some centralized deployment tool, such as MS Autopilot) but then we lack the tools inside Canonical to try, take screenshots etc as we are not a Windows software house :)

Reply:
I completely understand your position; however, this single line did not provide enough context to warrant its inclusion. How the approach outlined in the tutorial and the one suggested in this line are different with respect to scalability is not sufficiently clear (a lot is assumed of the reader). When I revise this PR I can look for ways to edit for clarity.

The questions you raise about doing a separate tutorial are very important. We need to think about this and some of the complexities that you highlight, but that conversation is outside the scope of this PR.

A good first start might be find that earlier version of the tutorial where you used the registry.